### PR TITLE
vim-patch:8.1.1279,8.2.{360,361}

### PR DESCRIPTION
**Pull request by janlazo**

### CODE REVIEW

File: ci/build.ps1 [hunk 1]
```diff
@@ -36,7 +36,8 @@ $scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh'
   Invoke-Expression $scoop
 }
 
-scoop install diffutils perl
+scoop install diffutils
```
Comment (janlazo) [inline ci/build.ps1]: http://repo.msys2.org/msys/x86_64/diffutils-3.5-1-x86_64.pkg.tar.xz is unavailable.
pacman downloads from the same site. How are the mingw builds passing?

File: ci/build.ps1 [hunk 2]
```diff
@@ -36,7 +36,8 @@ $scoop = (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh'
...
+scoop install perl
```
Comment (justinmk) [inline ci/build.ps1]: > ci/Appveyor: install perl separately

 What's the motivation ?

File: ci/build.ps1 (hunk 1 above)
Comment (justinmk) [reply ci/build.ps1]: maybe https://github.com/lukesampson/scoop/issues/2722 is relevant

File: ci/build.ps1 (hunk 2 above)
Comment (janlazo) [reply ci/build.ps1]: Yes.
scoop can install perl but not diffutils because of the issue with msys2.

**Comment by lacygoill** [comment]
Just for information, [8.2.0361](https://github.com/neovim/neovim/pull/11977/commits/b00650cfe909313e58aa8181474d7f016d9df94a) fixes a crash in Nvim.

File: src/nvim/eval.c [hunk 3]
```diff
@@ -8114,10 +8114,16 @@ void get_system_output_as_rettv(typval_T *argvars, typval_T *rettv,
...
+  int r = OK;
+
   if (arg->v_type == VAR_PARTIAL && arg->vval.v_partial != NULL) {
     callback->data.partial = arg->vval.v_partial;
     callback->data.partial->pt_refcount++;
     callback->type = kCallbackPartial;
+  } else if (arg->v_type == VAR_STRING
+             && arg->vval.v_string != NULL
+             && ascii_isdigit(*arg->vval.v_string)) {
+    r = FAIL;
```
Comment (justinmk) [inline src/nvim/eval.c]: seems really specific to only check for a number rather than "is this a valid function", but oh well...

**Comment by justinmk** [comment]
as a habit commits like https://github.com/neovim/neovim/pull/11977/commits/45b8dc0c3df861e5fa9ba67f8ac5e0993d2dba5d should say *why* we are  reverting it. But let's not block this.

**Pull request merged**